ex3.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def matrix_size(matrix) -> list[int, int]:
```

Comment:
matrix_size doesn;t represent what the function does. In numpy size represent the total amount of elements in the array, not the amount of rows or collumns.